.typos.toml

```diff
@@ -0,0 +1,15 @@
+[default]
+extend-ignore-re = [
+    "Lorem ipsum [a-zA-Z .,]*",
+]
+
+[default.extend-identifiers]
+FillStrat = "FillStrat" # short for strategy
```

Comment:
This probably isn't a great name, but fixing that is not the job of this pr

Reply:
Agreed.